examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_dynamic_list_input.py

```diff
@@ -0,0 +1,32 @@
+from promptflow import tool
+from typing import List, Union, Dict
+
+
+def dummy_list(prefix: str, **kwargs) -> List[Dict[str, Union[str, int, float, list, Dict]]]:
+    # dummy random.
+    import random
+
+    words = ["apple", "banana", "cherry", "date", "elderberry", "fig", "grape", "honeydew", "kiwi", "lemon"]
+    result = []
+    for i in range(10):
```

Comment:
It will contain repetitive items by using this dummy method. Let's avoid this. I think getting a random 1 ~ 10 total count is better than a fixed 10.